flowlayout/src/androidTest/AndroidManifest.xml

```diff
@@ -19,10 +19,8 @@
 
     <uses-sdk android:targetSdkVersion="30" />
 
-    <application>
-        <activity
-            android:name="com.google.accompanist.flowlayout.TestActivity"
-            android:theme="@android:style/Theme.Material.NoActionBar" />
+    <application android:theme="@android:style/Theme.Material.NoActionBar">
+        <activity android:name="androidx.activity.ComponentActivity" />
```

Comment:
Why is this necessary? If it's for compose-test I'm surprise this works here (normally it has to be added to debug/ with `debugImplementation("androidx.compose.ui:ui-test-manifest:$compose_version")`)

Reply:
It doesn't have to be in `debugImplementation`, but yeah we can use the manifest dependency instead.